src/condor_utils/file_transfer.cpp

```diff
@@ -6953,10 +6916,14 @@ FileTransfer::InvokeMultipleFileTransferPlugin( CondorError &e, int &exit_status
 					exit_status, label, error_message.c_str(), UrlSafePrint(transfer_url) );
 			}
 
-			SendPluginOutputAd( this_file_stats_ad );
+			// send the ad over the pipe, or just store it
+			if ( ! SendPluginOutputAd( this_file_stats_ad )) {
+				pluginResultAds.emplace_back(this_file_stats_ad);
+			}
```

Comment:
This looks weird, but I guess is OK (and it's good that it's a local and not a forked global).